### PR TITLE
Update hr.json

### PR DESCRIPTION
**Pull request by WebShapedBiz**
Few fixes

### CODE REVIEW

File: locales/hr.json [hunk 1]
```diff
@@ -3,21 +3,17 @@
...
         "apply": "Primjena",
         "type_address": "Unos adrese",
         "use_this_address": "Koristiti ovu adresu",
         "back_to_store": "Povratak u trgovinu",
         "close_cart": "Zatvoriti košaricu",
         "show": "Prikazivanje",
         "hide": "Skrivanje",
-        "apply_changes": "Primjena svih izmjena",
-        "yes_use_it": "Da, koristiti",
-        "save_changes": "Spremi izmjene",
```
Comment (dominic-blain) [inline locales/hr.json:17]: Please make sure you add back those phrases:

- save_changes
- back_to_orders
- change_password
- add

File: locales/hr.json [hunk 2]
```diff
@@ -42,64 +38,57 @@
...
-        "shipping_taxes_calculated_at_checkout": "Dostava i porezi će se obračunati na blagajni, nakon unosa svih podataka.",
+        "subtotal": "Neto iznos",
+        "shipping_taxes_calculated_at_checkout": "PDV i eventualni troškovi dostave obračunat će se nakon unosa svih podataka.",
         "loading": "Pripremamo vašu košaricu ...",
         "secured_by": "Secured by Snipcart",
-        "summary": "Pregled Narudžbe",
+        "summary": "Pregled narudžbe",
         "empty": "Vaša košarica je prazna.",
-        "invoice_number": "Broj Ponude",
-        "view_detailed_cart": "Detalji košarice"
```
Comment (dominic-blain) [inline locales/hr.json:52]: Please make sure you add back this phrase:

- view_detailed_cart

File: locales/hr.json [hunk 3]
```diff
@@ -42,64 +38,57 @@
...
+        "title": "Podaci o primatelju računa",
         "address": "Adresa na računu",
         "continue_to_shipping": "Nastaviti na dostavu",
         "use_different_shipping_address": "Koristiti drugu adresu za dostavu"
     },
     "customer": {
-        "information": "Podaci o kupcu"
-    },
-    "customer_details": {
-        "title": "Detalji o kupcu"
```
Comment (dominic-blain) [inline locales/hr.json:83]: Please keep "title" under "customer_details".

File: locales/hr.json [hunk 4]
```diff
@@ -42,64 +38,57 @@
...
         "loading": "Učitavam...",
-        "no_orders": "Nema Ponuda.",
-        "no_subscriptions": "Nema pretplata.",
-        "view_invoice": "Pregled Računa",
+        "no_orders": "Narudžbe nisu pronađene.",
+        "no_subscriptions": "Pretplate nisu pronađene.",
+        "view_invoice": "Pogledati račun",
         "sign_out": "Odjava",
-        "show_more_orders": "Više %{smart_count} Narudžbi |||| Više %{smart_count} Narudžbi",
-        "show_more_subscriptions": "Više %{smart_count} Pretplata |||| Više %{smart_count} Pretplata",
```
Comment (dominic-blain) [inline locales/hr.json:100]: Please make sure you add back those phrases:

- show_more_subscriptions
- orders
- subscriptions

File: locales/hr.json [hunk 5]
```diff
@@ -117,17 +106,10 @@
         "action": "Poslati email poruku za ponovno postavljanje lozinke"
     },
     "reset_password_form": {
-        "title": "Resetiranje lozinke",
```
Comment (dominic-blain) [inline locales/hr.json:120]: The structure here should be kept like this (with the correct translations, of course):
  ```
"reset_password_form": {
        "title": "Reset your password",
        "password": "New password",
        "confirmation_password": "Re-enter your new password",
        "action": "Reset password"
    },
    "change_password_form": {
        "title": "Change your password",
        "current_password": "Current password",
        "password": "New password",
        "confirmation_password": "Re-enter your new password",
        "action": "Change password"
    },
```

File: locales/hr.json [hunk 6]
```diff
@@ -184,19 +166,18 @@
...
-            "deferred_payment": "Kasnije plaćanje (po izdanoj Ponudi i sl.)"
+            "deferred_payment": "Plaćanje po ponudi"
         }
     },
     "discounts": {
         "title": "Popusti"
     },
     "cart_summary": {
         "taxes": "PDV",
-        "taxes_included_in_price": "Neke stavke imaju uključen PDV.",
```
Comment (dominic-blain) [inline locales/hr.json:195]: Please add back this phrase: taxes_included_in_price

File: locales/hr.json [hunk 7]
```diff
@@ -285,22 +266,21 @@
                 "description": "Nije bilo moguće autorizirati vaše plaćanje. Molimo pokušajte kasnije."
             }
         },
-        "customer_exists": "Ova se email adresa već koristi.",
-        "customer_password_missmatch": "Obje lozinke moraju biti jednake.",
-        "customer_password_token_expired": "Token za resetiranje lozinke je istekao.",
-        "customer_password_token_invalid": "Token za resetiranje lozinke nije valjan.",
-        "customer_not_found": "Email adresa nije registrirana u sustavu.",
-        "invalid_credentials": "Neispravna email adresa ili lozinka.",
-        "customer_current_password_invalid": "Neispravna lozinka."
```
Comment (dominic-blain) [inline locales/hr.json:294]: Please add back this phrase : customer_current_password_invalid

**Review by dominic-blain — CHANGES_REQUESTED** [review]
Hello there,

Thanks for taking the time to maintain this locale. There were still a few phrases missing. This can happen for a few reason, but the most probable one is that your local repository is not synced with ours.

A simple way to fix it would be to add them back manually (I commented on the missing lines) but a better way would be to [configure a remote for your fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork) and then [sync your fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork).

Hope this helps!

**Comment by WebShapedBiz** [comment]
Thank you very much Dominic for finding the time to help me out.

I'm not an expert in GitHub PRs and I could not figure out why are some fields missing since I did make a translation.
Nevertheless, I'll try to figure out how to fix this mess I'm causing.
Cheers.
Siniša Perović
Webshaped Business OÜ
***@***.*** ***@***.***)
+385 92 1244627 (tel:+385%2092%201244627)
https://webshaped.biz

**Comment by WebShapedBiz** [comment]
I've made all required changes and click "Commit changes". I hope that this time it will work.

Thank you again for your time and effort.

Cheers.

**Comment by dominic-blain** [comment]
Hey there!

It's my pleasure :) It seems however that nothing has changed since last time. After clicking "Commit changes", you need to "Push" those change to this branch. This will send your _local_ changes to your _remote_ repository.

image

Hope this helps! We are almost there :)

**Comment by WebShapedBiz** [comment]
Thank you for your time and effort but I'm completely lost now. I did everything you said I should do in your last post and at the end I've made a push.
1. I cannot understand why my first PR didn't have all translation included. My local file, from which I've copied everything was at the end of that first PR a different file from the pushed GitHub version. How is that possible?
2. The same goes for the second PR.
3. I can see that there are two PRs waiting with green checks. What are those if I didn't push anything? Where did my changes go? 
Slika zaslona 2021-04-14 14 45 12

Sorry for this and thank you again for your time and effort @dominic-blain, I won't be bothering you anymore. If there is really nothing changed in those two PR's then I'm declaring myself stupid and giving up this hell I don't understand.

Cheers.

**Comment by dominic-blain** [comment]
After comparing both PR, it looks like the PR #108 is the most up-to-date and contains all your fixes. #108 is based on your **master** branch, while this one (#120) is based on your branch **patch-2**. It is hard to know exactly what happened on your side but this may be the reason why this PR got out of sync.

I will close this PR and merge the other one. Feel free to create new PRs based on your master branch. It should be simpler to update and keep in sync this way!

Have a nice one!

**Pull request closed without merging**